services/server/package.json

```diff
@@ -56,6 +56,7 @@
     "@ethereum-sourcify/lib-sourcify": "^1.10.0",
     "@google-cloud/cloud-sql-connector": "1.4.0",
     "abitype": "1.0.6",
+    "aws-sdk": "^2.1691.0",
```

Comment:
I think you should use `@aws-sdk/client-s3` which is the newest version of the package.

Reply:
I'm already using it, but I probably forgot to install it. I'm wondering how can it work... But if you look at here, I'm actually using v3

https://github.com/ethereum/sourcify/blob/1676e81cd15b7ab5d8c5d0d26b86cec3e41835dc/services/server/src/server/services/storageServices/S3RepositoryService.ts#L5C1-L10C29

Reply:
Yeah saw that. I think it works because it is a dependency of the mock library.